### DISCUSSION TITLE
Add an option to let us use the muffled audio effect when transitioning to gameplay when the game is unfocused

### DISCUSSION THREAD

**Discussion by nyannoying1337** [ppy/osu#15192] [Ideas]
Would be nice to have the audio effect where it gets muffled when transitioning into gameplay while the game is unfocussed. its a pretty nice effect imo. not sure if if its 100% clear what i mean so i attach this video here https://streamable.com/nkg4bn

**Comment by peppy** [comment]
I think if we chose to do this it wouldn't be a toggle but always applied, probably in the same ratio to the inactive volume amount. I think it will work quite well, definitely interested to give it a try!

Before people ask, there will probably be a toggle game-wide to disable the effect for people that don't like it, if that becomes a thing. It won't be per usage.
